key-manager/src/main/scala/org/bitcoins/keymanager/WalletStorage.scala

```diff
@@ -166,28 +208,94 @@ object WalletStorage {
     encryptedEither
   }
 
+  /** Reads the raw unencrypted mnemonic from disk */
+  private def readUnencryptedMnemonicFromDisk(
+      seedPath: Path): CompatEither[ReadMnemonicError, DecryptedMnemonic] = {
+
+    val jsonE: CompatEither[ReadMnemonicError, ujson.Value] = {
+      if (Files.isRegularFile(seedPath)) {
+        val rawJson = Files.readAllLines(seedPath).asScala.mkString("\n")
+        logger.debug(s"Read raw mnemonic from $seedPath")
+
+        Try(ujson.read(rawJson)) match {
+          case Failure(ujson.ParseException(clue, _, _, _)) =>
+            CompatLeft(ReadMnemonicError.JsonParsingError(clue))
+          case Failure(exception) => throw exception
+          case Success(value) =>
+            logger.debug(s"Parsed $seedPath into valid json")
+            CompatRight(value)
+        }
+      } else {
+        logger.error(s"Mnemonic not found at $seedPath")
+        CompatLeft(ReadMnemonicError.NotFoundError)
+      }
+    }
+
+    import MnemonicJsonKeys._
+    import ReadMnemonicError._
+
+    val readJsonTupleEither: CompatEither[
+      ReadMnemonicError,
+      (Vector[String], Long)] = jsonE.flatMap { json =>
+      logger.trace(s"Read mnemonic JSON: $json")
```

Comment:
Will this log the unencrypted seed if the `TRACE` log level is set? If so, we should remove it. We don't want the mnemonic seed accidentally getting caught in logs, especially since this is unencrypted.